index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <header class="head">
+      <h2>Welcome</h2>
+      <img src="./Icons/Unionburger_icon.png" alt="" />
+    </header>
+    <section class="main">
+      <h2>
+        Hey there. I’m Yonas <br />
+        <span> I’m a software developer</span>
+      </h2>
+      <p>
+        I can help you build a product , feature or website. Look through some
+        of my work and experience! If you like what you see and have a project
+        you need coded, don’t hestiate to contact me.
+      </p>
+      <div class="icons">
+        <img
+          class="icon_btn"
+          src="Icons/social media icons/twitter.svg"
+          alt="twitter"
+        />
+
+        <img
+          class="icon_btn"
+          src="Icons/social media icons/Linkedin icon.svg"
+          alt="Linkedin"
+        />
+        <img
+          class="icon_btn"
+          src="Icons/social media icons/medium.svg"
+          alt="medium"
+        />
+        <img
+          class="icon_btn"
+          src="Icons/social media icons/github.svg"
+          alt="github"
+        />
+        <img
+          class="icon_btn"
+          src="Icons/social media icons/angellist.svg"
+          alt="angellist"
+        />
+      </div>
```

Comment:
These icons are a list of your social media profiles, so make sure to include them in a `ul` tag and have an `<a>` tag for each icon linked with your social media profile.